webcompat/static/js/lib/labels.js

```diff
@@ -24,9 +24,9 @@ issues.LabelsView = Backbone.View.extend({
     "click .js-LabelEditorLauncher.is-active": "closeEditor"
   },
   keyboardEvents: {
-    "l": "labelWrap"
+    "l": "labelWarp"
```

Comment:
I got the reference and liked the idea, but I also get the benefits of having a meaningful name. 
+1 on "good first bug"